css/style.css

```diff
@@ -1,3 +1,952 @@
+  }
+
+  .page__header {
+    max-width: 375px;
+  }
+
+}
+
+/* tablet 768 */
+
+@media screen and (min-width:768px) {
+
+  .page {
+    max-width: 768px;
+  }
+
+  .page__header {
+    max-width: 768px;
+    height: 113.54px;
+  }
+
+  .header__logo-mobile {
+    width: 156.18px;
+    height: 33.74px;
+    left: 43px;
+    top: 41.51px;
+  }
+
+  .menu__burger {
+    width: 60.85px;
+    height: 5px;
+    top: 46.49px;
+    right: 44px;
+    border-top: 8px solid #000;
+    border-bottom: 8px solid #000;
+  }
+
+  .new-collection__goto {
+    top: 0;
+    width: 100%;
+    height: 415.28px;
+    margin: 0 auto 67.43px auto;
+  }
+
+  .new-collection__goto .goto__img-mini {
+    display: none;
+  }
+
+  .new-collection__goto .goto__img-tablet {
+    display: block;
+    width: 100%;
+    height: 415.28px;
+  }
+
+  .new-collection__goto .goto__title {
+    left: 44.42px;
+    font-size: 50px;
+    line-height: 48px;
+  }
+  
+  .goto__shop {
+    font-size: 20px;
+  }
+  
+  .goto__arrow {
+    width: 70.72px;
+  }
+
+  .products__title {
+    font-size: 30px;
+  }
+  
+  .products__list {
+    height: 352px;
+    justify-content: space-around;
+  }
+  
+  /* rasta */
+  
+  .product-rasta_r {
+    right: 75px;
+    bottom: 25px;
+    width: 20px;
+    height: 20px;
+  }
+  
+  .product-rasta_b {
+    right: 50px;
+    bottom: 25px;
+    width: 20px;
+    height: 20px;
+  }
+  
+  .product-rasta_g {
+    right: 25px;
+    bottom: 25px;
+    width: 20px;
+    height: 20px;
+  }
+  
+  .product-rasta_y {
+    right: 0px;
+    bottom: 25px;
+    width: 20px;
+    height: 20px;
+  }
+
+ /* camel */
+
+ .products__product-camel {
+   display: block;
+   align-self: flex-start;
+   text-align: left;
+   position: relative;
+   width: 280px;
+   height: 340px;
+ }
+
+ .product-camel_g {
+   position: absolute;
+   display: inline-block;
+   right: 75px;
+   bottom: 25px;
+   background: #c4c4c4;
+   border-radius: 50%;
+   width: 20px;
+   height: 20px;
+ }
+
+ .product-camel_o {
+   position: absolute;
+   display: inline-block;
+   right: 50px;
+   bottom: 25px;
+   background: #e29d69;
+   border-radius: 50%;
+   width: 20px;
+   height: 20px;
+ }
+
+ .product-camel_r {
+  position: absolute;
+  display: inline-block;
+  right: 25px;
+  bottom: 25px;
+  background: #b62e3c;
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+}
+
+ .product-camel_y {
+   position: absolute;
+   display: inline-block;
+   right: 0px;
+   bottom: 25px;
+   background: #fdb431;
+   border-radius: 50%;
+   width: 20px;
+   height: 20px;
+ }
+
+  .products__img {
+    width: 229.52px;
+    height: 233.42px;
+  }
+  
+  .products__name {
+    font-size: 20px;
+    line-height: 19px;
+  }
+  
+  .products__cost {
+    font-size: 25px;
+    line-height: 24px;
+  }
+
+}
+
+/* desktop 1024 */
+
+@media all and (min-width: 1024px) {
+
+  .page {
+    max-width: 1024px;
+  }
+
+  .page__header {
+   max-width: 1024px;
+   justify-content: center;
+   align-content: center;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)